include/deal.II/base/memory_space_data.h

```diff
@@ -118,6 +123,8 @@ namespace MemorySpace
     // This is not used but it allows to simplify the code until we start using
     // CUDA-aware MPI.
     std::unique_ptr<Number[]> values_dev;
+
+    std::vector<ArrayView<const Number>> values_sm;
```

Comment:
is this also "currently not used"? If yes, add a comment please.

Reply:
It is used on host.